pkg/base1/cockpit.js

```diff
@@ -3630,8 +3630,8 @@ function factory() {
          * close function to only work when a problem is
          * indicated.
          */
-        var old_close;
         if (shared) {
+            var old_close = client.close;
```

Comment:
Please use `const` here -- scoping of `var` is really broken in JS, it does not do what you think it does; it always applies to the scope of the containing function, not just the `{}` block

Reply:
done, thanks.

I explicitly avoided changing this for exactly this reason.  I knew it was different, and I didn't know which one I wanted :rofl: